### DISCUSSION TITLE
github for students - Can I use university email as form of proof ?

### DISCUSSION THREAD

**Discussion by dmendo77** [community/community#135458] [GitHub Education]
### Hi everyone,

In order to obtain free license of Github Copilot is required a student ID or Form of proof. Recently, I was enrolled in a virtual master degree of Spain University where did not provide me any of those documents required to apply free license.

Could I use as form of proof an email (with no letterhead) received directly of the University where state the date of my enrollment?

**Comment by davevad93** [comment]
Hi @dmendo77 ,

An educational email is required to apply for the `Student Developer Pack`, but you must be currently enrolled in an academic program, the requirements to apply are the following:

- ***Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.***
- ***Have a verifiable school-issued email address or upload documents that prove your current student status.***
- ***Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).***
- ***Be at least 13 years old.***

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) also state the following:
-  ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

Therefore, if your academic institution doesn't provide you with an ID, you can use one of the above documents to submit your application.
